pdr_backend/sim/sim_engine.py

```diff
@@ -139,6 +117,10 @@ def run_one_iter(self, test_i: int, mergedohlcv_df: pl.DataFrame):
         X_train, X_test = X[st_:fin, :], X[fin : fin + 1, :]
         ytran_train, _ = ytran[st_:fin], ytran[fin : fin + 1]
 
+        high_value, low_value, _ = data_f.get_highlowclose(
+            mergedohlcv_df, predict_feed, testshift
+        )
+
```

Comment:
- Change `get_highlowclose()` -> `get_highlow()` since it doesn't need close price

Rename:
- `high_value` -> `cur_high`
- `low_value` -> `cur_low`
- `curprice` -> `cur_close`
- `true_price` -> `next_close`
- `predprice` -> `pred_next_close`